.eslintrc.js

```diff
@@ -27,16 +25,14 @@ module.exports = {
     }
   },
   "extends": [
-
     "eslint:recommended",
     "plugin:jsx-a11y/recommended", // require("eslint-plugin-jsx-a11y")
     "plugin:mozilla/recommended", // require("eslint-plugin-mozilla")
     "plugin:mozilla/browser-test",
     "plugin:mozilla/mochitest-test",
     "plugin:mozilla/xpcshell-test",
-    "prettier",
-    "prettier/react",
     "plugin:prettier/recommended",
+    "prettier/react",
```

Comment:
Let's put the comments like the others to be clearer on where these are coming from / are actually using dependencies:
```
    "plugin:prettier/recommended", // require("eslint-plugin-prettier")
    "prettier/react", // require("eslint-config-prettier")
```

Reply:
Done.